regress/pesterTests/FileBasedLogging.tests.ps1

```diff
@@ -0,0 +1,237 @@
+If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
+Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+Import-Module OpenSSHUtils -Force
+$tC = 1
+$tI = 0
+$suite = "FileBasedLogging"
+Describe "Tests for admin and non-admin file based logs" -Tags "CI" {
+    BeforeAll {
+        if($OpenSSHTestInfo -eq $null)
+        {
+            Throw "`$OpenSSHTestInfo is null. Please run Set-OpenSSHTestEnvironment to set test environments."
+        }
+        
+        $testDir = "$($OpenSSHTestInfo["TestDataPath"])\$suite"
+        if( -not (Test-path $testDir -PathType Container))
+        {
+            $null = New-Item $testDir -ItemType directory -Force -ErrorAction SilentlyContinue
+        }        
+
+        $sshLogName = "test.txt"
+        $sshdLogName = "sshdlog.txt"
+        $server = $OpenSSHTestInfo["Target"]
+        $opensshbinpath = $OpenSSHTestInfo['OpenSSHBinPath']
+        $nonadminusername = $OpenSSHTestInfo['NonAdminUser']
+        $adminusername = $OpenSSHTestInfo['AdminUser']
+        $password = $OpenSSHTestInfo['TestAccountPW']
+        $port = 47003  		
+        Remove-Item -Path (Join-Path $testDir "*$sshLogName") -Force -ErrorAction SilentlyContinue
+
+        <# Setup sshd_config file#>
+        
+        $sshdconfig_ori = Join-Path $Global:OpenSSHTestInfo["ServiceConfigDir"] sshd_config
+        $sshdconfig_custom = Join-Path $Global:OpenSSHTestInfo["ServiceConfigDir"] sshd_config_custom
+        if (Test-Path $sshdconfig_custom) {
+            Remove-Item $sshdconfig_custom -Force
+        }
+        Copy-Item $sshdconfig_ori $sshdconfig_custom
+        get-acl $sshdconfig_ori | set-acl $sshdconfig_custom
+        $content = Get-Content -Path $sshdconfig_custom
+        $newContent = $content -replace "Subsystem	sftp	sftp-server.exe -l DEBUG3", "Subsystem	sftp	sftp-server.exe -l DEBUG3 -f LOCAL0"
+        $newContent | Set-Content -Path $sshdconfig_custom
+				   
+        #skip when the task schedular (*-ScheduledTask) cmdlets does not exist
+        $ts = (get-command get-ScheduledTask -ErrorAction SilentlyContinue)
+        $skip = $ts -eq $null
+        if(-not $skip)
+        {
+            Stop-SSHDTestDaemon   -Port $port
+        }
+        if(($platform -eq [PlatformType]::Windows) -and ([Environment]::OSVersion.Version.Major -le 6))
```

Comment:
Did you run / manually test your changes on windows 7?

Reply:
Not yet. I'll do that next.